template/static/index.js

```diff
@@ -0,0 +1,4 @@
+function handle_form()
+{
+  alert("Submit button pressed");
```

Comment:
You are not using file anywhere. Do you need it?

Reply:
yes, the file index.js was not used anywhere, so there is no need of it

Reply:
And also remove the pycahe

Reply:
Can you delete this then?

Reply:
yes, i delete this one